spec/rubocop/cop/lint/return_in_void_context_spec.rb

```diff
@@ -80,4 +80,12 @@ def self.initialize
       RUBY
     end
   end
+
+  context 'with no method' do
```

Comment:
Here, too: "when `return` is in top scope".

Reply:
Guess we can also have some tests with procs, lambda and a return in say class definition. Guess we didn't think of all of them in the past.

Reply:
Good point!